assets/js/post_office_search.tsx

```diff
@@ -1,8 +1,9 @@
 import React from 'react';
 import { createRoot } from 'react-dom/client';
+// @ts-ignore
```

Comment:
Not ideal, but necessary until `@18f/identity-address-search` includes type definitions in the published package.